internal/cli/organizations.go

```diff
@@ -408,9 +415,12 @@ func deleteOrganizationCmd(cli *cli) *cobra.Command {
 		Use:   "delete",
 		Args:  cobra.MaximumNArgs(1),
 		Short: "Delete an organization",
-		Long:  "Delete an organization.",
-		Example: `auth0 orgs delete
-auth0 orgs delete <id>`,
+		Long: "Delete an organization.\n\n" +
+			"To delete interactively, use `auth0 orgs delete` with no arguments.\n\n" +
+			"To delete non-interactively, supply the organization id and the `--force` " +
+			"flag to skip confirmation.",
+		Example: `  auth0 orgs delete
+  auth0 orgs delete <id>`,
```

Comment:
Consider adding the `auth0 orgs delete <id> --force` example in a future PR, like it is in other commands.